stdlib/public/runtime/Demangle.cpp

```diff
@@ -25,7 +25,10 @@ _buildDemanglingForNominalType(Demangle::Node::Kind boundGenericKind,
                  typeBytes + sizeof(void*) * description->GenericParams.Offset);
     for (unsigned i = 0, e = description->GenericParams.NumPrimaryParams;
          i < e; ++i, ++genericParam) {
-      typeParams->addChild(_swift_buildDemanglingForMetadata(*genericParam));
+      auto demangling = _swift_buildDemanglingForMetadata(*genericParam);
```

Comment:
Only on Linux with an Optional of MetadataKind::Opaque. I didn't investigate further as _swift_buildDemanglingForMetadata() suggests this is expected.